shared/cluster.go

```diff
@@ -409,3 +409,139 @@ func WriteDataPod(namespace string) (string, error) {
 
 	return RunCommandHost(cmd)
 }
+
+type ServiceType string
+
+const (
+	Server ServiceType = "server"
+	Agent  ServiceType = "agent"
+)
+
+func (s ServiceType) IsValidServiceType() bool {
+	valid := []ServiceType{Server, Agent}
+	for _, i := range valid {
+		if s == i {
+			return true
+		}
+	}
+
+	return false
+}
+
+type ServiceAction string
+
+const (
+	Stop    ServiceAction = "stop"
```

Comment:
Are these part of systemctl commands?

Reply:
yes

Reply:
Then suggest naming them to be in sync with the cli command, like `SystemctlAction`

Reply:
Or may be even better to have something similar to what we have for `KubectlCommand` func here: https://github.com/rancher/distros-test-framework/blob/main/shared/cluster.go#L139

Reply:
Similary we can have a `ServiceCommand` func for the k3s/rke2 services

Reply:
fixed and moved to product.go